### PR TITLE
CLOUDP-73721: mongocli ops-manager admin backup blockstore(s) update

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-73721

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

```
./bin/mongocli om admin backup blockstore update test --assignment  --uri mongodb://localhost:29022 -o json -P ops
{
  "id": "test",
  "uri": "mongodb://localhost:29022",
  "ssl": false,
  "assignmentEnabled": true,
  "encryptedCredentials": false,
  "usedSize": 204800,
  "loadFactor": 1
}
```

```
./bin/mongocli om admin backup blockstore update test --assignment  --uri mongodb://localhost:29022  -P ops
Blockstore configuration 'test' updated.
```

### CODE REVIEW

File: internal/cli/admin_opts.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 MongoDB Inc
```
Comment (gssbzn) [inline internal/cli/admin_opts.go:1]: [nit] can we move this to the `/opsmanager/admin/backup` package? this way the name is less confusing, `Admin` is to broad of a term to be in the global `cli` package

**Review by gssbzn — CHANGES_REQUESTED** [review]
one small nit to move the AdminOpts closer to where we need it

File: internal/cli/admin_opts.go (hunk 1 above)
Comment (andreaangiolillo) [reply internal/cli/admin_opts.go:1]: In this way we are going to hit `import cycle not allowed`. Should I rename it as `create_opts`?

Comment (gssbzn) [reply internal/cli/admin_opts.go:1]: feel free then to create a `backupstore` package in the admin one

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**